NuGet.Config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
The RC3 packages will be coming into cli-deps tonight - it was only bringing "rc2" packages up until now.